doc/usecases/draft/copy_paste_between_own_maps.md

```diff
@@ -0,0 +1,38 @@
+# Use Case: Copy & Paste of Selection Between a User's Own Maps
+
+## Summary
+
+- **Scope:** All Layers, except Base
+- **Level:** User Goal
+- **Actors:** App User
+- **Brief:** The user can copy and paste a selection of elements, including succeeding crops, between his own maps.
```

Comment:
Do we really need to distinguish own and other maps? I don't see how this differs in the use case description or in the implementation?